src/porepy/viz/data_saving_model_mixin.py

```diff
@@ -45,16 +45,37 @@ class DataSavingMixin:
 
     def save_data_time_step(self) -> None:
         """Export the model state at a given time step, and log time."""
```

Comment:
Please provide a short explanation of the main options:
None - export all
list - export if time in list. If list is empty, no times are exported.